pwa/src/components/attributes/attribuutForm.tsx

```diff
@@ -214,6 +215,13 @@ export const AttributeForm: React.FC<AttributeFormProps> = ({attributeId, entity
                         <Spinner/>
                       ) : (
                         <div>
+                          {saveSpinner === true ? (
+                            <div className="spinner-overlay">
+                              <div className="spinner-overlay-content">
+                                <Spinner />
+                              </div>
+                            </div>
```

Comment:
Right now, we are creating the following boilerplate every time we want to include this component:

```
<div className="spinner-overlay">
  <div className="spinner-overlay-content">
    <Spinner /> // only this line might not be considered boilerplate
  </div>
</div>
```

I would say that this piece of code is a componenten of itself. Right now, when we need to make changes to this component, we would have to make changes everywhere we've included it like this.

A better way of structuring this component would be to:

1. Make a folder in de `/components` folder (named something like `loadingOverlay`, depending on how you're going to call your component)
2. Create your component here with the `.tsx` and `.css` files (see feedback on styling to understand why we need a `.css` file here)
3. Lastly, instead of writing the boilerplate code everywhere, it can become something like:

```
{ saveSpinner && <LoadingOverlay /> } // this is your current state name
{ loading && <LoadingOverlay /> } // i would suggest something like this
```